mlir/lib/Conversion/SPIRVToLLVM/SPIRVToLLVM.cpp

```diff
@@ -1591,6 +1649,9 @@ void mlir::populateSPIRVToLLVMConversionPatterns(
 
       // Return ops
       ReturnPattern, ReturnValuePattern>(patterns.getContext(), typeConverter);
+
+  patterns.add<GlobalVariablePattern>(clientAPIForAddressSpaceMapping,
```

Comment:
I think you could concatenate this add operation to the patters above:
```
    patters.add<...>.add<...>()

Reply:
That'd involve reformatting the whole `add<...>` block. I'm not sure it's worth it. Are you OK with the current form?

Reply:
Yes I am OK with the current form.